### DISCUSSION TITLE
3Blue1Brown{YTR}

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#495] [General]
https://www.youtube.com/c/3blue1brown/videos
這個頻道不錯喔 有講述不少AI的觀念 講得蠻易懂的

舉個例子：
數字辨識：https://www.youtube.com/watch?v=aircAruvnKk&ab_channel=3Blue1Brown

**Comment by brian431** [comment]
確實是個很好的頻道，不過我發過了歐

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by RogelioKG** [comment]
3Blue1Brown的優點在於，他很擅長將原本很抽象的概念具體化與可視化，
讓你看到一個數學式子背後的意義，
或者是說它真正的「模樣」是長什麼樣子，
這些參數開始變動後又是長什麼樣子。
By the way 他的微積分系列和線代系列也很棒，
連同機器學習系列我都看完了😌。

**Comment by williamyen0623** [comment]
現在在youtube上的教學頻道不在少數
要找到自己需要的內容還需要額外花費時間
謝謝版主的分享～

**Comment by Ben6uo** [comment]
youtube總是那麼的多好用的東西 雖然要用不少時間來找

**Comment by HsuChen88** [comment]
圖像化難懂的觀念能讓更多人了解任何知識其中的涵義，希望能讓更多人受益。

**Comment by Yilin20021016** [comment]
這個AI影片對我來說之後會有很多地方用到 很好
